frontend/public/global.css

```diff
@@ -3,6 +3,7 @@
   --main-fg-color: white;
   --selected-bg-color: #86e3ed;
   --comment-color: #eb8e5b;
+  --highlight-color: #dc4e47;
```

Comment:
Why is this highlight color added here when it is overridden by the original two lines below?